src/schema/ghga.yaml

```diff
@@ -511,7 +511,7 @@ classes:
         description: >-
           One or more attributes that further characterizes this Library Preparation Protocol.
       target regions:
-        required: true
+        required: false
```

Comment:
General remark: Don't forget to update the version of the schema on the top of the file.